qemu/tests/driver_in_use.py

```diff
@@ -97,15 +99,15 @@ def run_subtest(sub_type):
 
     env["bg_status"] = 0
     run_bg_flag = params.get("run_bg_flag")
-    sub_type = params["sub_test"]
+    main_test = params["main_test"]
```

Comment:
Not suggest renaming this param, because many other cases are using it.
```
$ grep driver_in_use -nr
qemu/tests/cfg/netkvm_in_use.cfg:2:    type = driver_in_use
qemu/tests/cfg/driver_load_stress.cfg:2:    type = driver_in_use
qemu/tests/cfg/viorng_in_use.cfg:2:    type = driver_in_use
qemu/tests/cfg/win_virtio_driver_update_test.cfg:2:    type = driver_in_use
qemu/tests/cfg/balloon_in_use.cfg:2:    type = driver_in_use
qemu/tests/cfg/rng_driver_load.cfg:15:        - driver_in_use:
qemu/tests/cfg/virtio_storage_in_use.cfg:2:    type = driver_in_use
```

Reply:
Thanks.
change "sub_test" back in the viorng_in_use.cfg file,and keep "main_test" in driver_in_use.py.